deployment/solana/common/operations_token.go

```diff
@@ -0,0 +1,164 @@
+package deployment_solana_common
+
+import (
+	"context"
+
+	solana_go "github.com/gagliardetto/solana-go"
+	solRpc "github.com/gagliardetto/solana-go/rpc"
+	solCommomUtil "github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
+	solTokenUtil "github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/tokens"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/solana"
+	deployment_solana "github.com/smartcontractkit/chainlink/deployment/solana/extension"
+)
+
+type DeploySolanaTokenInput struct {
+	TokenProgramName string
+	TokenDecimals    uint8
+}
+
+var DeploySolanaTokenOp = deployment.NewOperation(
```

Comment:
These Ops replicates the logic of the changesets defined in [ccip/solana](https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/solana/cs_solana_token.go)